internal/bft/requestpool.go

```diff
@@ -72,6 +73,7 @@ type Pool struct {
 type requestItem struct {
 	request []byte
 	timeout *time.Timer
+	begin   time.Time
```

Comment:
Can you give a more descriptive name? 
```suggestion
	additionTimestamp   time.Time
```

Reply:
fix